lib/auth/init.go

```diff
@@ -648,6 +668,54 @@ func createPresets(ctx context.Context, rm PresetRoleManager) error {
 	return nil
 }
 
+// PresetUserManager contains the required User Management methods to
+// create a preset User.
+type PresetUserManager interface {
```

Comment:
Nit: I think we try to avoid XXXManager and other "Java-esques" in the code.
```suggestion
// PresetUsers contains the required User Management methods to
// create a preset User.
type PresetUsers interface {
```